device/ios/Plugin/DevicePlugin.swift

```diff
@@ -39,6 +39,7 @@ public class DevicePlugin: CAPPlugin {
             "model": modelName,
             "operatingSystem": "ios",
             "osVersion": UIDevice.current.systemVersion,
+            "iOSVersion": Double(UIDevice.current.systemVersion) ?? 0,
```

Comment:
If the iOS Version is something like 16.4.1 this will return 0. We should probably return something more meaningful in that case. Like 16.41 or just 16 or 1641 as an integer.

Reply:
One idea:
```
let number = "16.2.1"
let majorVersion = number.split(separator: ".")
let majorVersionInt = Int(majorVersion[0])
``